client/landing/jetpack-cloud/sections/scan/history/index.jsx

```diff
@@ -75,6 +75,19 @@ const ScanHistoryPage = ( {
 		[ dispatchRecordTracksEvent, siteId, siteSlug ]
 	);
 
+	const clickHeader = React.useCallback(
```

Comment:
@rcanepa introduced a `useTrackCallback` hook in #42819. It might be worth it reusing it.

Reply:
@monsieur-z - Good catch! I'll revisit this code once that's merged.